bluepymm/prepare_combos/parse_files.py

```diff
@@ -128,10 +128,10 @@ def read_mm_recipe_yaml(recipe_filename):
     mecombos = pandas.DataFrame(columns=["layer", "fullmtype", "etype"])
     for region in recipe['neurons']:
         for etype in region['traits']['etype'].keys():
-            n_combos = len(mecombos)
-            mecombos.loc[n_combos, 'layer'] = region['traits']['layer']
-            mecombos.loc[n_combos, 'fullmtype'] = region['traits']['mtype']
-            mecombos.loc[n_combos, 'etype'] = etype
+            gid = len(mecombos)
```

Comment:
is this a gid ?

Reply:
I guess this is rather 'end'?

Reply:
Yes, I don't know how to call/code that. Let's go for end, it's better.